hardhat.config.ts

```diff
@@ -472,7 +472,7 @@ export default {
   etherscan: {
     customChains: [
       {
-        network: 'zkemv',
+        network: 'zkevm',
```

Comment:
How did that ever work then? Is it not using the custom chain? Does it work if we just delete that network?